src/core/env/prod/terraform.tfvars

```
@@ -97,7 +97,7 @@ app_gateway_management_certificate_name = "management-platform-pagopa-it"
 app_gateway_wisp2_certificate_name      = "wisp2-pagopa-it"
 app_gateway_wisp2govit_certificate_name = "wisp2-pagopa-gov-it"
 app_gateway_kibana_certificate_name     = "kibana-platform-pagopa-it"
-app_gateway_wfespgovit_certificate_name = ""
+app_gateway_wfespgovit_certificate_name = "wfesp-pagopa-gov-it"
 app_gateway_min_capacity                = 8 # 5 capacity=baseline, 8 capacity=high volume event, 15 capacity=very high volume event
```

```suggestion
app_gateway_min_capacity                = 5 # 5 capacity=baseline, 8 capacity=high volume event, 15 capacity=very high volume event
```
from capacity metric, we can decrease app_gateway_min_capacity to baseline